src/Applications/LDAS_App/GEOSldas_LDAS.rc

```diff
@@ -59,6 +59,8 @@ LSM_CHOICE: 1
 #    1 :  bilinear interpolation (default)
 #    0 :  nearest neighbor
 #
+#   for adas-coupling exp, forcing in CS grid 
+#   set  MET_HINTERP: 0 
```

Comment:
MET_HINTERP=1 may still make sense with COUPLING=1 (if forcing is from lat/lon GEOS data).   
Added comment into issue #296 to address checking for MET_HINTERP=0 when GEOS forcing is on cube-sphere grid.

Reply:
changed the information text